ui/tests/acceptance/managed-namespace-test.js

```diff
@@ -31,8 +33,11 @@ module('Acceptance | Enterprise | Managed namespace root', function(hooks) {
   });
 
   test('it shows the managed namespace toolbar when feature flag exists', async function(assert) {
+    await logout.visit();
     await visit('/vault/auth');
-    assert.equal(currentURL(), '/vault/auth?namespace=admin&with=token', 'Redirected to base namespace');
+    console.log(currentURL(), 'current');
```

Comment:
is this `console.log` left here intentionally?

Reply:
Nope! Thanks :)